lang/language-variants-0.20.6

```diff
@@ -1,6 +1,5 @@
 apex
 bash
-c-sharp
```

Comment:
Is this related to adding the new submodule?

Reply:
No. I made a mistake last week when building this list. It's because the c-sharp was failing to build in CI but it was due to a lack of memory, not an incompatibility with the latest tree-sitter.